### PR TITLE
Add Ndabaga Opinion hook-up in jssandbox

### PR DESCRIPTION
**Pull request by imsickofmaps**
Uses same model as Articles but on API for `/ndabaga/`

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review but easier after landing #8 as merges from there included

File: directory/api.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        """
+        Modifying the data to provide the Categories and Sub Categories.
+        Structure is {dir: {cat: {sub_cat{c: "a", c2: "b", "c3: "c"}}}}
+        This function handles /api/category/
+        """
+        if isinstance(data_dict, dict):
+            if 'meta' in data_dict:
+                del(data_dict['meta'])
+
+            if data_dict["objects"] != []:
```
Comment (smn) [inline directory/api.py]: `if data_dict["objects"]:`

File: directory/api.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        sub_cat_obj_data = sub_category_bundle.data
+                        for key, value in sub_cat_obj_data.iteritems():
+                            if key in ["content_1", "content_2", "content_3"]:
+                                sub_cat_content[key] = value
+                        sub_category_dict[sub_cat_obj_data["name"]] = sub_cat_content
+
+                    directory[category_name] = sub_category_dict
+                data_dict['directory'] = directory
+
+            del (data_dict['objects'])
```
Comment (smn) [inline directory/api.py]: you guys really love brackets!

File: directory/tests/test_api.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+class TestCategoryApi(TestCase):
+    fixtures = ['test_directory.json']
+
+    def test_correct_url(self):
+        url = reverse('api_dispatch_list',
+                      kwargs={'resource_name': 'category',
+                      'api_name': 'api'})
+        response = self.client.get(url)
+        self.assertEqual(response.request["PATH_INFO"], "/api/category/")
```
Comment (smn) [inline directory/tests/test_api.py]: remove pls?

**Comment by smn** [comment]
Getting js test failures: https://travis-ci.org/praekelt/go-nike-ghr/jobs/9011426, `self.im` doesn't seem to be defined.

**Pull request merged**